Update blog; Added the R in Africa blog (#90)
forwards/website_source: content/blog/2019-11-15-the-state-of-r-in-africa.md

---

> title: The-state-of-R-in-Africa

Reviewer: ```suggestion
title: The state of R in Africa
```

Reviewer: spaces are needed here as this is the title used in the blog post

> banner:"/images/blog/The-state-of-R-in-Africa/banner.jpg"

Reviewer: ```suggestion
banner: "/images/blog/The-state-of-R-in-Africa/banner.jpg"
```

Reviewer: A space is needed here for the YAML to parse properly

author: Lorna Maria
date: '2019-11-15'
description: "2019 Round-up of initiatives in the R community across Africa."
slug: the-state-of-r-in-africa
categories: ["news"]
tags: ["community"]
tocify: no
---

> ##Growth of the R ecosystem in Africa 

Reviewer: ```suggestion
## Growth of the R ecosystem in Africa 
```

Reviewer: spaces are needed after `##` otherwise the title is not recognised when the markdown is rendered.

Over the past 2 years, we have seen R transform from just an academic and health care research language to one of the major languages in the budding data science community in Africa. For a long time, data scientists and related disciplines have relied on R and many other packages to analyse data, however, the broader developer community didn’t fully understand its power in the tech industry.

> An effective model to enter the Africa developer ecosystem relies on a small group of enthusiasts creating a safe learning environment, documenting and sharing their learning experiences and slowly attracting interest learners. Africa R users group are using this model to introduce R as a language for data science by organising communities across different cities in Africa.

Reviewer: ```suggestion
An effective model to enter the Africa developer ecosystem relies on a small group of enthusiasts creating a safe learning environment, documenting and sharing their learning experiences and slowly attracting interested learners. Africa R users group are using this model to introduce R as a language for data science by organising communities across different cities in Africa.
```

##Evolution of chapters and representation

> The growth of technical talent in Africa positively correlates with the growth of communities around certain technologies/tools/languages. These communities are creating a long table to share opportunities on the continent. When we shift the spotlight to R in the Africa tech community we see an evolution useRs groups of not only academic researchers completing their PhD or master’s degrees but also people using R at their workplaces or for their freelance projects. 

Reviewer: ```suggestion
The growth of technical talent in Africa positively correlates with the growth of communities around certain technologies/tools/languages. These communities are creating a long table to share opportunities on the continent. When we shift the spotlight to R in the Africa tech community we see an evolution in useR groups of not only academic researchers using R in their PhD or master’s degrees but also people using R at their workplaces or for their freelance projects. 
```

There is a unique representation in the Africa R community that shows how the useRs have created a safe space for equal active participation. Global communities like R Ladies and Women in Machine Learning and Data Science have given so many women in the community a platform to start R user groups in their localities. The support from these groups has contributed to the early success of R user groups as the safest places to start your data science career.
![R Ladies at the Annual Women Techmakers Event](/images/blog/The-state-of-R-in-Africa/representation.jpg)

> Some of the chapters include [Kampala](https://mobile.twitter.com/Rladieskampala),[Lagos](https://mobile.twitter.com/LagosRUsers),[Dar es salaam](https://mobile.twitter.com/daR_users), [Abidjan](https://mobile.twitter.com/AbidjanRusers), [Johannesburg](https://mobile.twitter.com/RLadiesJozi),[DRC] (https://mobile.twitter.com/DrcUsers),[Cotonou](https://mobile.twitter.com/RLadiesCtn) among others. 

Reviewer: Perhaps make clear which are general R User Groups and which are R-Ladies groups?

> ##Events

Reviewer: ```suggestion
## Events
```

Monthly Meetups: These are the most popular events in the community. Groups of 10-50 useRs meet every month to learn and help each other with R related issues. They are usually supported by the local useR group in the area and are at the forefront of the growing popularity of data science in the local communities.

> SatRday: The R community in Johanessburg and Kampala have both hosted satRday events in April 2019 and May 2019 respectively. These events featured light talks and hands-on workshops for R users and were free. These have been the biggest R focused events on the continent for the year and hope they will spread to other regional hosts in the coming year. 

Reviewer: ```suggestion
SatRday: The R community in Johanessburg and Kampala have both hosted satRday events in April 2019 and May 2019 respectively. These events featured lightning talks and hands-on workshops for R users and were free. These have been the biggest R focused events on the continent for the year and the hope is they will spread to other regional hosts in the coming year. 
```

![A useR meetup in Dar es salaam](/images/blog/The-state-of-R-in-Africa/events.jpg)

> ##The Future

Reviewer: ```suggestion
## The Future
```

> R as a statistical programming language has a great value to the data science community on the continent and useRs to showing this value through their work, research and education. Most useR groups in Africa currently focus on developing material that shows the amazing facets of R and renders it as an option on the table for potential clients and the community.We are starting to see R cross paths with other programming languages thus raising a need to create room for it in learning and early career choices through building communities, mentoring talent and eventually powering businesses.

Reviewer: ```suggestion
R as a statistical programming language has a great value to the data science community on the continent and useRs are showing this value through their work, research and education. Most useR groups in Africa currently focus on developing material that shows the amazing facets of R, rendering it as an option on the table for potential clients and the community. We are starting to see R cross paths with other programming languages thus raising a need to create room for it in learning and early career choices through building communities, mentoring talent and eventually powering businesses.
```

> Follow the [AfricaR](https://mobile.twitter.com/AfricaRUsers)  and individual chapters on their social media to view the most recent updates on events.

Reviewer: ```suggestion
Follow [AfricaR](https://mobile.twitter.com/AfricaRUsers)  and individual chapters on their social media to view the most recent updates on events.
```
